### ISSUE TITLE
Public exposure of what should be a private accessor

### ISSUE THREAD

**Issue by scepticalcat** [KarthickNarayana/toy_robot#5]
https://github.com/KarthickNarayana/toy_robot/blob/3d570a92d5c8b027e2456ec4f7bce47223719ecf/lib/toy_robot/cli.rb#L8

Given the application has no need to access the simulator through the CLI, I would have expected `attr_reader` to have been declared privately.

**Comment by KarthickNarayana** [comment]
Yes simulator is accessed in cli.rb. This has to be private.   There is no need of it to go out of cli.
